online-grocery-store/src/components/layout/MainLayout.js

```diff
@@ -8,7 +9,19 @@ const Header = () => {
       <Grid.Col span={2} />
       <Grid.Col span={8}>
         <Flex h="100%" justify="space-between" align="center">
-          <Link href="/">Header</Link>
+          <Link href="/">
+            <Image radius="md" src="favicon.ico" h={40} w="auto" />
```

Comment:
Should also have the Website name after the logo as well. And please use another logo.

Reply:
fixed